kernel/aster-nix/src/syscall/arch/x86.rs

```diff
@@ -214,6 +216,8 @@ impl_syscall_nums_and_dispatch_fn! {
     SYS_ARCH_PRCTL = 158       => sys_arch_prctl(args[..2], &mut context);
     SYS_CHROOT = 161           => sys_chroot(args[..1]);
     SYS_SYNC = 162             => sys_sync(args[..0]);
+    SYS_MOUNT = 165            => sys_mount(args[..5]);
+    SYS_UMOUNT = 166           => sys_umount2(args[..2]);
```

Comment:
See: https://github.com/torvalds/linux/blob/v6.9/arch/x86/entry/syscalls/syscall_64.tbl#L177

So it should be `SYS_UMOUNT2 = 166           => sys_umount(args[..2]);`

Reply:
Thank you for pointing out the discrepancy. I have made the necessary correction to align with the latest Linux kernel naming conventions:
```rust
SYS_UMOUNT2 = 166 => sys_umount(args[..2]);
```
The change has been implemented accordingly.